middleware/markdown/process.go

```diff
@@ -151,7 +163,7 @@ func defaultTemplate(c Config, metadata Metadata, requestPath string) []byte {
 	title := metadata.Title
 	if title == "" {
 		title = filepath.Base(requestPath)
-		if body, _ := metadata.Variables["markdown"].([]byte); len(body) > 128 {
+		if body, _ := metadata.Variables["body"].([]byte); len(body) > 128 {
```

Comment:
On L70, it looks like we're storing that as a string already. Type assertion to []byte?